torch/distributed/rpc.py

```diff
@@ -96,6 +99,14 @@ def init_rpc(name, backend='pg'):
         raise RuntimeError("Unrecognized RPC backend ", backend)
 
 
+def get_id(workerName=None):
+    _check_initialized()
```

Comment:
nit: A decorator would be a nice way to do this for all functions.

Reply:
Good point, let me add that.

Reply:
Python always uses snake case
```
def get_id(worker_name=None):
```